kb/licensing/how-is-mbedtls-protected.md

```diff
@@ -2,18 +2,12 @@
 
 ## Copyright
 
-The copyright in the source code primarily protects Mbed TLS (formerly known as PolarSSL). We own all the copyright in the source code. For code that is contributed to us, we request the original author to share the copyright with us.
+The copyright in the source code primarily protects Mbed TLS (formerly known as PolarSSL). We own all the copyright in the source code. For code that is contributed to us, we request the original author to grant copyright to The Mbed TLS Contributors.
```

Comment:
```suggestion
Mbed TLS (formerly known as PolarSSL) is, like most software, protected by copyright.
It is licensed for use under an open-source Apache 2.0 licence.
The *project* is owned by TrustedFirmware.org, and
the code is owned by all the people and companies who have contributed to it, collectively known as *The Mbed TLS Contributors*.
When contributing code to us, the author is required to sign off all commits with a [Developer Certificate of Origin](../../dco.txt) line,
confirming that the code submitted can (legally) become part of the project, and be subject to the same Apache 2.0 licence.
```

Reply:
Done except for re-paragraphing and US-English spelling of "license" to match existing text.

Reply:
This is much better I think, thanks @tom-cosgrove-arm. However - not all the code is owned by "The Mbed TLS Contributors" (i.e. Everest). I suggest:

The *project* is owned by TrustedFirmware.org, and the code is owned by all the people and companies who have contributed to it, collectively known as *The Mbed TLS Contributors* (except for some files in the 3rdparty directory which have copyright as noted in the file headers).